django_toolkit/fallbacks/circuit_breaker/rules.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+import abc
+import logging
+
+logger = logging
+
+
+class Rule(object):
+    __metaclass__ = abc.ABCMeta
+
+    def __init__(
+        self,
+        max_failures,
```

Comment:
I think `max_failures` don't need to be in the abstract `Rule` constructor, as each implementation should will have its own conditionals to open the CB.